src/test_l3_dsHdmiIn.c

```diff
@@ -0,0 +1,1461 @@
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_get_allmstatus(void)
+{
+    gTestID = 13;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret   = dsERR_NONE;
+    int select = 0;
+    dsHdmiInPort_t port = dsHDMI_IN_PORT_NONE;
+    bool allmstatus = false;
+    uint8_t numInputPorts = 0;
+
+    numInputPorts = UT_KVP_PROFILE_GET_UINT8("dsHdmiIn/numberOfPorts");
+    printf("\n**********Please select port***********\n");
+    scanf("%d", &select);
+    readAndDiscardRestOfLine(stdin);
+
+    if(select <= 0 || select >numInputPorts)
+    {
+      printf("\n invalid port selected\n");
+      goto exit;
+    }
+    port = (dsHdmiInPort_t)select;
+
+    ret = dsGetAllmStatus(port , &allmstatus);
+    if(ret != dsERR_NONE)
+    {
+       UT_LOG_ERROR("Result: Failed dsGetAllmStatus Error:[%s]",ut_control_GetMapString(dsError_mapTable, ret));
+    } else {
+           UT_LOG_INFO("Result : Passed dsGetEdidVersion port type: %s, portnumber: %s\t"
+                         "retrieved allm status: %s",
+                 ut_control_GetMapString(dsHdmiInPort_mapTable, port), port,
+                 ut_control_GetMapString(bool_mapTable, allmstatus));
+    }
+exit:
+UT_LOG_INFO("Out %s", __FUNCTION__);
+}
+
+/**
+* @brief This test to get supported game features list.
+*
+* This test function is to get supported game features list of HdmiInput port on platform.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 014@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_get_gamefeatureslist(void)
+{
+    gTestID = 14;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret   = dsERR_NONE;
+    dsSupportedGameFeatureList_t features_list;
+
+    ret = dsGetSupportedGameFeaturesList( &features_list);
+    if(ret != dsERR_NONE)
+    {
+       UT_LOG_ERROR("Result: Failed dsGetSupportedGameFeaturesList Error:[%s]",ut_control_GetMapString(dsError_mapTable, ret));
+    } else {
+           UT_LOG_INFO("Result : Passed dsGetSupportedGameFeaturesList retrieved features_list count: %d",
+                 features_list.gameFeatureCount);
+    }
+    UT_LOG_INFO("Out %s", __FUNCTION__);
+}
+
+/**
+* @brief This test to get AV latency.
+*
+* This test function is to get AV latency of HdmiInput port on platform.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 015@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_get_avlatency(void)
+{
+    gTestID = 15;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret   = dsERR_NONE;
+    int audio_latency = 0, video_latency = 0;
+   
+    ret = dsGetAVLatency(&audio_latency , &video_latency);
+    if(ret != dsERR_NONE)
```

Comment:
Remove the else case

In all cases, you need to see the results output

Reply:
done